assets/js/modules/pagespeed-insights/components/dashboard/DashboardPageSpeed.js

```diff
@@ -229,28 +229,30 @@ export default function DashboardPageSpeed() {
 			const allAudits = select(
 				MODULES_PAGESPEED_INSIGHTS
 			).getAuditsWithStackPack( referenceURL, strategy, 'wordpress' );
+
 			if ( ! allAudits || ! Object.keys( allAudits ).length ) {
 				return [];
 			}
 
-			const audits = [];
-			Object.keys( allAudits ).forEach( ( auditSlug ) => {
-				const audit = allAudits[ auditSlug ];
-				if (
-					( audit.scoreDisplayMode !== 'numeric' &&
-						audit.scoreDisplayMode !== 'binary' ) ||
-					audit.score >= 0.9
-				) {
-					return;
-				}
-
-				audits.push( {
-					id: audit.id,
-					title: audit.title,
-				} );
-			} );
-
-			return audits;
+			return Object.keys( allAudits )
+				.filter(
+					( auditSlug ) =>
+						allAudits[ auditSlug ].scoreDisplayMode ===
+							'metricSavings' && allAudits[ auditSlug ].score < 1
+				)
+				.sort( ( a, b ) => {
+					// If the scores are the same, sort alphabetically by
+					// audit slug. This is how the API returns audits.
+					if ( allAudits[ a ].score === allAudits[ b ].score ) {
+						return a.localeCompare( b );
```

Comment:
Hmm - actually, I don't think this is an appropriate case for using `localeCompare()`.

Note that the results of `localCompare()` [vary between languages](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare#using_locales) (locales). So it's possible that in some locales we'd end up sorting the list in a different order to that returned by the API.

I think in this scenario a simple string comparison would be appropriate e.g. 
```suggestion
						return a < b ? -1 : 1;
```

Reply:
Excellent spot, updated, thank you Tom!